pinot-core/src/main/java/org/apache/pinot/core/query/reduce/ExecutionStatsAggregator.java

```diff
@@ -363,19 +362,20 @@ public void setStats(@Nullable String rawTableName, BrokerResponseNative brokerR
 
   public void setStageLevelStats(@Nullable String rawTableName, BrokerResponseStats brokerResponseStats,
       @Nullable BrokerMetrics brokerMetrics) {
-    setStats(rawTableName, brokerResponseStats, brokerMetrics);
-
-    brokerResponseStats.setNumBlocks(_numBlocks);
-    brokerResponseStats.setNumRows(_numRows);
-    brokerResponseStats.setStageExecutionTimeMs(_stageExecutionTimeMs);
-    brokerResponseStats.setStageExecWallTimeMs(_stageExecEndTimeMs - _stageExecStartTimeMs);
-    brokerResponseStats.setStageExecutionUnit(_stageExecutionUnit);
```

Comment:
shouldn't stage execution start/end time be set? as well as the num row/blocks?

Let's rename this to operator execution time instead if that's confusing

Reply:
This is when tracing is disabled right? You mentioned that we only want wall time in that case. The rest should be filled only when tracing is enabled.

OperationExecution time is not correct name right? This is aggregated stage time.

Reply:
hmm i guess i didn't make my question clear let me rephase